ocs_ci/ocs/resources/pod.py

```diff
@@ -1210,6 +1210,22 @@ def upload(pod_name, localpath, remotepath, namespace=None):
     run_cmd(cmd)
 
 
+def download(pod_name, remotepath, localpath, namespace=None):
```

Comment:
Maybe we should use a more descriptive name? 
`download_file_from_pod`/`copy_file_from_pod`

Reply:
+1